lib/generators/solidus/auth/install/install_generator.rb

```diff
@@ -18,7 +18,7 @@ def generate_devise_key
         end
 
         def add_migrations
-          run 'bundle exec rake railties:install:migrations FROM=solidus_auth_devise'
+          run 'bundle exec rake railties:install:migrations FROM=solidus_auth'
```

Comment:
A next iteration on this generator could be to use the [`rake` command of Rails generators](https://api.rubyonrails.org/classes/Rails/Generators/Actions.html#method-i-rake), preferably with the `abort_on_failure` options set to `true`, so we do not continue with the installation if something fails while copying the migrations.

Also we could use the "magic" rake task that Railties generates for engines instead of using the `FROM` env var.

```rb
rake "solidus_auth:install:migrations", abort_on_failure: true
```

Reply:
Thanks @tvdeyen. PR incoming